services/movie.py

```diff
@@ -0,0 +1,30 @@
+from db.models import Movie, Genre, Actor
+
+
+def get_movies(genres_ids: list[int], actors_ids: list[int]) -> Movie:
```

Comment:
The return type hint for `get_movies` is incorrect. It should be `QuerySet[Movie]` instead of `Movie` because the function returns a queryset of movies, not just one.